Makefile

```diff
@@ -45,15 +45,19 @@ fetch: $(BUILD_DIR)/fetch
 fetch-reset:
 	@rm -f $(BUILD_DIR)/fetch
 
-$(BUILD_DIR)/fetch: $(BUILD_DIR)/ruby scripts/release.rss.groovy scripts/fetch-examples scripts/fetch-external-resources | $(OUTPUT_DIR)
```

Comment:
Deleted `scripts/fetch-examples`.  The same thing is now done via `scripts/fetch-external-resources`.